multichain-testing/test/support.ts

```diff
@@ -68,11 +68,14 @@ const makeKeyring = async (
   return { setupTestKeys, deleteTestKeys };
 };
 
-export const commonSetup = async (t: ExecutionContext) => {
+export const commonSetup = async (
+  t: ExecutionContext,
+  config = `../${process.env.FILE || 'config.yaml'}`,
```

Comment:
oh. this is why I couldn't find any way that `chainInfo` could vary. ambient access to `process.env`.

```suggestion
  {
    env,
    config = `../${env.FILE || 'config.yaml'}`,
  }
```

or, per #10038 ...

```suggestion
  {
    env = process.env,
    config = `../${env.FILE || 'config.yaml'}`,
  }
```

but I'd still pass `process.env` in explicitly from the script.